### PR TITLE
Added additive option for applying pose item

### PR DESCRIPTION
**Pull request by stzhang123**
It works by adding the value to the current attribute value, and use blend to modify the added amount; Added the clearcache argument in load method to allow artist to apply the additive repeatedly without having to unselect and reselect the pose item.

### CODE REVIEW

**Comment by krathjen** [comment]
Thanks @stzhang123!

The changes look great! I'll try and test this feature over the next day or two.

Cheers,

Kurt

**Comment by stzhang123** [comment]
Thank you so much Kurt! Feel free to reach out if you have any questions.

Best,

Steven Zhang

File: src/studiolibrarymaya/poseitem.py [hunk 1]
```diff
@@ -312,6 +313,13 @@ def loadSchema(self):
                 "default": False,
                 "persistent": True,
             },
+            {
```
Comment (krathjen) [inline src/studiolibrarymaya/poseitem.py:316]: Do you mind moving the `additive` option under the `mirror` option?

**Comment by krathjen** [comment]
Hey @stzhang123,

It's working well. The only thing I found when testing on a simple scene was the behavior for boolean attributes. The scene contained a single sphere with one pose having translations in X and the other pose having translations in Y. When middle mouse blending below `-50%` on either pose the sphere would disappear. I guess it makes sense, however, I wonder if we should just ignore boolean attributes for the additive option?

Thanks again.

Kurt

**Comment by stzhang123** [comment]
Hi Krathjen,

That's a good point! I haven't thought of bool attributes at all. Thank you
for mentioning that. I did some tweaks and pushed a new version just now.
Feel free to modify it if you have any suggestions!

Thank you!

Steven Z

**Comment by krathjen** [comment]
Hey @stzhang123,

It looks good. Thank you!

Feel free to merge your changes, otherwise, I'm happy to merge them.

If you do merge your changes just update the version to `2.8.0` in `src/studiolibrary/__init__.py`.

We really appreciate your help!

Cheers,

Kurt

**Pull request merged**